src/services/oathkeeper/index.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  OathkeeperUnauthorizedServiceError,
+  OathkeeperError,
+  OathkeeperForbiddenServiceError,
+  UnknownOathkeeperServiceError,
+} from "@domain/oathkeeper/errors"
+import axios from "axios"
+
+// import oathkeeper from "@ory/oathkeeper-client"
+
+// note: decisions is the admin API
+const decisionsPrefix = "http://oathkeeper:4456/decisions/"
```

Comment:
note for me and @bodymindarts 

this will need to be set to the correct domain name